src/Voyager.php

```diff
@@ -257,7 +257,10 @@ public function setting($key, $default = null)
     public function image($file, $default = '')
     {
         if (!empty($file)) {
-            return str_replace('\\', '/', Storage::disk(config('voyager.storage.disk'))->url($file));
+            $url = Storage::disk(config('voyager.storage.disk'))->url($file);
+            $url = str_replace('\\', '/', $url);
```

Comment:
This can be simplified by using an array as the needle-parameter like `$url = str_replace(['\\', '%5C'], '/', $url);`